docs/source/notebooks/AutoEncoders.md

```diff
@@ -73,99 +76,109 @@ print(x_test.shape)
 
 ## 3. Define Keras Model
 
-We will be defining a very simple autencoder. We define _three_ model building methods:
+We will be defining a very simple autencoder. We define _three_ model architectures:
 
-1. One to build a full end-to-end autoencoder.
-2. One to create a model that includes only the encoder portion.
-3. One that creates a model that includes only the decoder portion.
+1. An encoder: a series of densly connected layers culminating in an "output" layer that determines the encoding dimensions.
+2. A decoder: takes the output of the encoder as it's input and reconstructs the original data.
+3. An autoencoder: a chain of the encoder and decoder that directly connects them for training purposes.
 
 The only variable we give our model is the encoding dimensions, which will be a hyperparemter of our final transformer.
 
-```python
-from tensorflow import keras
+The encoder and decoder are views to the first/last layers of the autoencoder model. They'll be directly used in predict/inverse_transform/etc, so let's create some SciKeras models with those layers in {encoder, decoder}_model_.
+We will build these models within `_keras-build_fn` so that we can store our encoders and decoders as seperate models and call them from `transform` and `inverse_transform` respectively. 
```

Comment:
```suggestion
We will build these models within `_keras-build_fn` so that we can store our encoders and decoders as seperate models and call them from `transform` and `inverse_transform` respectively. 
```
```suggestion
We will build these models within `_keras_build_fn` so that we can store our encoders and decoders as separate models and call them from `transform` and `inverse_transform` respectively. 
```

Reply:
Sorry, I had un-pushed changes, apologies for the crosstalk. The wording now reads:

```
The encoder and decoder are views to the first/last layers of the autoencoder model.
They'll be directly used in ``transform`` and ``inverse_transform``, so we'll create some SciKeras models with those layers
and save them as in `encoder_model_` and `decoder_model_`. All three models are created within `_keras_build_fn`.
```

I think this is pretty consistent with your previous suggestion? Sorry again for forgetting to push that.

Reply:
Yeah, that all looks good